modules/tensor_mechanics/doc/content/documentation/systems/Modules/TensorMechanics/GlobalStrain/GlobalStrainAction.md

```diff
@@ -0,0 +1,28 @@
+# Global Strain Action
+
+!syntax description /Modules/TensorMechanics/GlobalStrain/GlobalStrainAction
+
+This action simplifies the input file syntax for global strain calculation required for maintaining strain periodicity. It also generates the auxiliary displacement field created by the global strain. It creates following MOOSE objects,
+
+### UserObject
+
+- [Global Strain UserObject](/GlobalStrainUserObject.md): Calculates the residual and jacobian corresponding to the scalar variable
+
+### ScalarKernel
+
+- [Global Strain ScalarKernel](/GlobalStrain.md): Solves for the global strain in terms of the scalar variable
+
+### Material
+
+- [Global Strain Material](/ComputeGlobalStrain.md): Calculates the global strain components from the scalar variable
+
+### AuxVariables
+
+- [Global Displacement AuxKernels](/GlobalDisplacementAux.md): Creates AuxVariables for each additional displacement components
```

Comment:
Do you really want to link to the kernel here, too?

Reply:
The documentation for how to create aux variables for visualization is within `GlobalDisplacementAux.md`. So, I wanted to provide same link. The text should say `AuxVariables` though.